libs/database/src/lib/entities/delayed-job.entity.ts

```diff
@@ -1,7 +1,22 @@
-import { AllowNull, AutoIncrement, Column, Default, ForeignKey, Index, Model, PrimaryKey, Table } from "sequelize-typescript";
+import {
+  AllowNull,
+  AutoIncrement,
+  Column,
+  Default,
+  ForeignKey,
+  Index,
+  Model,
+  PrimaryKey,
+  Table
+} from "sequelize-typescript";
 import { BIGINT, BOOLEAN, INTEGER, JSON, STRING, UUID } from "sequelize";
 import { User } from "./user.entity";
 
+interface JobMetadata {
+  entity_id: string; // or number, depending on your model's entity ID type
+  entity_type: string;
+  entity_name: string; // Name of the related entity
+}
```

Comment:
I don't think we should try to strongly type this. Future jobs could put anything they want in `metadata`, and we wouldn't want to have to update the schema here each time that happens. The `payload` below simply uses `object`, and that should work fine for this aw well.

Reply:
changed to object